jobs/src/main/scala/dcos/metronome/history/impl/JobHistoryServiceActor.scala

```diff
@@ -1,12 +1,13 @@
 package dcos.metronome
 package history.impl
 
-import akka.actor.{ ActorLogging, ActorRef, Props, Actor }
+import java.time.Clock
+
+import akka.actor.{ Actor, ActorLogging, ActorRef, Props }
 import dcos.metronome.behavior.{ ActorBehavior, Behavior }
 import dcos.metronome.history.JobHistoryConfig
 import dcos.metronome.model._
-import dcos.metronome.repository.{ Repository, LoadContentOnStartup }
-import dcos.metronome.utils.time.Clock
```

Comment:
we moved from `dcos.metronome.utils.time.Clock` -> `java.time.Clock` 
and we have `SettableClock`
is there value in having a clock we can mock?

Reply:
`SettableClock` implements `java.time.Clock`, does it not? (If not, it should, this is what we do in Marathon)